### PR TITLE
mriqc v0.3.3

### PR DESCRIPTION
**Pull request by jenreiter**

### CODE REVIEW

File: gears/flywheel/mriqc.json [hunk 1]
```diff
@@ -12,12 +12,22 @@
...
+      "description": "Save all outputs generated by MRIQC algorithm. These results will be saved in one single zip file.",
+      "type": "boolean"
     }
   },
   "custom": {
-    "docker-image": "flywheel/mriqc:v0.3.2"
+    "docker-image": "flywheel/mriqc:v0.3.3"
   },
-  "description": "MRIQC extracts no-reference IQMs (image quality metrics) from structural (T1w and T2w) and functional MRI (magnetic resonance imaging) data. Note, this gear only supports the generation of individual scan reports; group reports are not generated.",
+  "description": "MRIQC extracts no-reference IQMs (image quality metrics) from structural (T1w and T2w) and functional MRI (magnetic resonance imaging) data. Note, this gear only supports the generation of individual scan reports; group reports are not generated. Also note, for the auto-detection config option to work for this gear, the follow gears must be run beforehand: (1) dicom-mr-classifier then (2) dcm2niix (version > v0.3.1).",
```
Comment (lmperry) [inline gears/flywheel/mriqc.json]: (2) dcm2niix (version 0.3.1 or higher )

Comment (jenreiter) [reply gears/flywheel/mriqc.json]: Thanks for this feedback. This is fixed now.

**Comment by jenreiter** [comment]
Note, I am adding this [issue](https://github.com/flywheel-apps/mriqc/issues/17) into the scope of v0.3.3

**Comment by jenreiter** [comment]
@lmperry This PR is now ready for review/approval

**Pull request merged**